server/routerlicious/packages/lambdas/src/nexus/connect.ts

```diff
@@ -67,6 +70,7 @@ enum ConnectDocumentStage {
 	TokenExpirySet = "TokenExpirySet",
 	MessageClientConnected = "MessageClientConnected",
 	SocketTrackerAppended = "SocketTrackerAppended",
+	SessionTrackerAppended = "SessionTrackerAppended",
```

Comment:
Nit: it would be good to document what the individual flags mean here. Or at least the new one, while we're adding it.